graylog2-server/src/main/java/org/graylog2/utilities/GRNRegistry.java

```diff
@@ -39,6 +39,7 @@
     // TODO find a way to unify these
     private static final ImmutableSet<GRNType> BUILTIN_TYPES = ImmutableSet.<GRNType>builder()
             .add(GRNType.create("collection", "collections:"))
+            .add(GRNType.create("capability", "capabilities:"))
```

Comment:
does this need a permissions prefix? capabilities will never be entities that are managed with permissions.
Same goes for roles.
I'm not even sure we should be using GRNs for that, actually..

Reply:
Not sure, but I thought it also didn't hurt at the moment. I also don't see a reason *not* to use GRNs for that. What's your thinking behind that?

Reply:
To clarify: My thinking was, that we might need to allow modification of capabilities (e.g. adding new permissions from plugins).

Now that I spelled it out... Extending capabilities will probably an internal thing without user interaction, so we also don't need permissions for that. :smile:

Reply:
> Not sure, but I thought it also didn't hurt at the moment. I also don't see a reason _not_ to use GRNs for that. What's your thinking behind that?

Just for simplicity. Currently we could be having three easily readable constants in the Grants table: `View`, `Manage`, `Own`

Reply:
> To clarify: My thinking was, that we might need to allow modification of capabilities (e.g. adding new permissions from plugins).
> 
> Now that I spelled it out... Extending capabilities will probably an internal thing without user interaction, so we also don't need permissions for that. smile

having capabilities to modify capabilities :see_no_evil:  reaching tourning completeness :-D

Reply:
> Just for simplicity. Currently we could be having three easily readable constants in the Grants table: `View`, `Manage`, `Own`

Okay, sounds good. So I would update the code to only use simple keywords `view`, `manage` and `own` for the capabilities. If we ever need to use GRNs, it should be a simple migration. :+1: